CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+# Wrapping FluentUI Apple Controls in FluentUI React Native
+
+## Pre-requisites
+
+This guide assumes you:
+
+- Are familiar with Objective C / Swift programming
+- Have read through the [React Native Docs](https://reactnative.dev/docs/). In particular:
+  - Understand classes vs function components (we use the latter) and [hooks](https://reactjs.org/docs/hooks-intro.html). Here's a good [video](https://www.youtube.com/watch?v=dpw9EHDh2bM) that explains function components and hooks for traditional OOP developers.
+  - Understand [Native Modules](https://reactnative.dev/docs/native-modules-ios).
+- Have a local fork of FluentUI React Native and have run the test app.
+
+## Random info related to FluentUI React Native needed for this guide
+
+Here I list a lot random concepts / info about specifics to FluentUI React Native. This is by no means that organized, and I'm happy for this info to be split out / moved around.
```

Comment:
feels like this is TMI

Reply:
Took this out, and worded it slightly better.